bw2io/extractors/simapro_csv.py

```diff
@@ -285,11 +395,84 @@ def get_project_name(cls, data):
 
     @classmethod
     def invalid_uncertainty_data(cls, amount, kind, field1, field2, field3):
+        """
+        Determine if the uncertainty data is invalid.
+
+        Parameters
+        ----------
+        amount : str
+            The amount of uncertainty.
+        kind : str
+            The kind of uncertainty.
+        field1 : str
+            The first field of uncertainty data.
+        field2 : str
+            The second field of uncertainty data.
+        field3 : str
+            The third field of uncertainty data.
+
+        Returns
+        -------
+        bool
+            `True` if the uncertainty data is invalid, `False` otherwise.
+
+        Notes
+        -----
+        This method checks if the given uncertainty data is invalid based on the kind of uncertainty.
```

Comment:
...same here: https://github.com/brightway-lca/brightway2-io/pull/176/files#r1157014630

Reply:
Incorporated Notes in Extended Description.